File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    Integer passengers;

    public UberVan(
        String license, 
        Account driver, 
        Map<String, Map<String,Integer>> typeCarAccepted, 
        ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }

    public UberVan(String license, Account driver) {
        super(license, driver);
    }

    @Override
    public void setPassengers(Integer passengers) {
        if(passengers == 6)
            this.passengers = passengers;
        else
        System.out.println("Necesita ingresar 6 pasajeros");
    }

}

File: Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial
    
    def setPassengers(self, passengers):
        if passengers == 6:
            self._passengers = passengers
        else:
            print('Necesita ingresar 6 pasajeros')